### PR TITLE
Added right-click event handler.

### PR DESCRIPTION
**Pull request by damian-codilime**
- test added
- docs modified

### CODE REVIEW

**Comment by TrySound** [comment]
Is it Table only feature?

**Comment by damian-codilime** [comment]
I don't see any `onClick` events in other components both docs and sources.

**Comment by bvaughn** [comment]
Yes, this is a `Table` only feature. Both `List`, `Grid`, `Masonry`, etc defer rendering of the outer cell to application code (so you can declare whatever event handlers you want directly on the rendered content).

File: docs/Table.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ This component expects explicit `width` and `height` parameters.
 | onHeaderClick | Function |  | Callback invoked when a user clicks on a table header. `({ columnData: any, dataKey: string, event: Event }): void` |
 | onRowClick | Function |  | Callback invoked when a user clicks on a table row. `({ event: Event, index: number, rowData: any }): void` |
 | onRowDoubleClick | Function |  | Callback invoked when a user double-clicks on a table row. `({ event: Event, index: number, rowData: any }): void` |
+| onRowRightClick | Function |  | Callback invoked when a user right-clicks on a table row. `({ event: Event, index: number, rowData: any }): void` |
```
Comment (bvaughn) [inline docs/Table.md]: Thanks for updating docs too! 😄 

nit: Docs should be alpha-sorted

File: source/Table/Table.jest.js [hunk 2]
```diff
@@ -766,6 +766,27 @@ describe("Table", () => {
...
+      const rows = rendered.querySelectorAll(".ReactVirtualized__Table__row");
+      Simulate.contextMenu(rows[0]);
+      Simulate.contextMenu(rows[3]);
+      expect(onRowRightClick).toHaveBeenCalledTimes(2);
+      expect(onRowRightClick.mock.calls.map(call => call[0].index)).toEqual([
+        0,
+        3
+      ]);
+    });
+  });
```
Comment (bvaughn) [inline source/Table/Table.jest.js]: Thanks for adding a test 😁

File: source/Table/defaultRowRenderer.js [hunk 3]
```diff
@@ -12,14 +12,21 @@ export default function defaultRowRenderer({
   key,
   onRowClick,
   onRowDoubleClick,
+  onRowRightClick,
```
Comment (bvaughn) [inline source/Table/defaultRowRenderer.js]: nit: alpha-sort

**Comment by bvaughn** [comment]
Thanks for the PR! My only requests were alpha-sorting nits so I went ahead and made them myself 😄

**Comment by bvaughn** [comment]
This feature has been released in version [9.10.0](https://github.com/bvaughn/react-virtualized/blob/master/CHANGELOG.md#9100). Thank you for contributing!

**Pull request merged**